### PR TITLE
Fix make test_template targets

### PR DESCRIPTION
**Pull request by t0yv0**
I'm not entirely sure what the go test is doing by default to find the list of templates, but it's not scanning the folder. So when adding new templates make test_tempate target is broken. This works around it.

### CODE REVIEW

**Review by kpitzen — APPROVED** [review]

**Comment by t0yv0** [comment]
@stack72 you had some comments in chat giving the background on why we have the current logic that selects only specific tests to run. But I wasn't sure if this change is unsafe in some way? It seems to me this is still a safe addition and it helps test things locally..

**Comment by cnunciato** [comment]
This doesn't seem to cause my tests to run locally, but I'm not sure it's supposed to. Should we merge it now to see if it results in the tests being run in CI?

**Comment by t0yv0** [comment]
This PR does not affect CI but it does unblock running tests locally before they are checked in (as form your branch) via:

```
make test_template.<template-name>
```

This was waiting on Paul Stack review though, I think this is not how we want to keep things. I'll just close for now.

**Pull request closed without merging**
